hack/README.md

```diff
@@ -0,0 +1,88 @@
+
+## Script overview
+
+| Script | Description |
+| :-- | :-- |
+| [tekton_in_kind.sh](#tekton_in_kind.sh) | Stands up a K8s cluster using the [kind](https://kind.sigs.k8s.io/) tool and deploys Tekton `pipeline`, `triggers` and `dashboard` components |
+| [tekton_ci.sh](#tekton_ci.sh) | Sets up a GitHub webhook to a fork of the `tekton/plumbing` repo. using using the `smee` tool. |
+
+See [DEVELOPMENT.md](https://github.com/tektoncd/plumbing/blob/main/DEVELOPMENT.md) for complete complete usage examples.
+
+---
+
+## Script details
+
+### tekton_in_kind.sh
+
+This script uses uses the [`kind` tool](https://kind.sigs.k8s.io/) to create a local K8s cluster in Docker and then deploys [Tekton Pipeline](https://github.com/tektoncd/pipeline), [Tekton Triggers](https://github.com/tektoncd/triggers) and [Tekton Dashboard](https://github.com/tektoncd/dashboard) components, into it.
+
+#### Installation and prerequisites
+
+- `go`: go 1.14+
+- `kubectl`: Install the K8s CLI *(see [Install tools](https://kubernetes.io/docs/tasks/tools/))*
+- `docker`: Install Docker *(see [Get Docker](https://docs.docker.com/get-docker/))*
+- `kind`: Install `kind` *(see ["quick start" documentation](https://kind.sigs.k8s.io/docs/user/quick-start/))*
+
+#### Usage
+
+```sh
+tekton_in_kind.sh [-c cluster-name -p pipeline-version -t triggers-version -d dashboard-version]
+```
+> **Note**: the default `cluster-name` is `'tekton'`
+
+#### Internals
+
+The script, after using `kind` to create the K8s cluster, will then use `kubectl` to install the `latest` released versions of Tekton components unless other versions are specified on the optional arguments. Here is a snippet of how the script does this using `kubectl`:
+
+```bash
+# Use `-p` arg. value or `latest`
+kubectl apply -f https://storage.googleapis.com/tekton-releases/pipeline/previous/${TEKTON_PIPELINE_VERSION}/release.yaml
+# Use `-t` arg. value or `latest`
+kubectl apply -f https://storage.googleapis.com/tekton-releases/triggers/previous/${TEKTON_TRIGGERS_VERSION}/release.yaml
+kubectl apply -f https://storage.googleapis.com/tekton-releases/triggers/previous/${TEKTON_TRIGGERS_VERSION}/interceptors.yaml || true
+# Use `-d` arg. value or `latest`
+kubectl apply -f https://github.com/tektoncd/dashboard/releases/download/${TEKTON_DASHBOARD_VERSION}/tekton-dashboard-release.yaml
+```
+
+> **Note**: `kind cluster create`  automatically creates a K8s context named `'kind-tekton'` and and makes it current for `kubectl` commands.
+
+#### cleanup
+
+If you wish to delete the cluster that the script created, use the following command:
+
+```shell
+kind delete cluster --name tekton
+```
+
+The `kind` tool will also use the cluster name from the `KIND_CLUSTER_NAME` environment variable if set.
+
+```shell
+export KIND_CLUSTER_NAME=tekton
+$ kind delete cluster
```

Comment:
We haven't included the prompt string (`$`) in any of the other shell blocks.  Suggest either removing it here or adding it in the other code blocks.